docs/tooling/pepsi.md

```diff
@@ -25,26 +25,26 @@ The simulation is paused automatically until the binary starts.
 
 This command does the following:
 
-- checks if a toolchain is installed, downloads, and installs one if necessary
-- builds the code for the NAO target
-- uploads binary, configuration, motion files, neural networks, etc. to the NAO(s)
-- restarts HULK service on the NAO(s)
+-   checks if a toolchain is installed, downloads, and installs one if necessary
```

Comment:
Your formatter introduces inconsistent formatting of bullet points. We should decide for one style and be consistent.

Reply:
I now reverted to the original style, I think it is the more intuitive one especially if one doesn't use a formatter.